topics/list_topics.go

```diff
@@ -0,0 +1,35 @@
+package topics
+
+import (
+	"context"
+	"regexp"
+
+	kafkago "github.com/segmentio/kafka-go"
+)
+
+// ListTopics returns a slice of all the Topics
+func ListTopics(ctx context.Context, client *kafkago.Client) (topics []kafkago.Topic, err error) {
```

Comment:
Currently the function would be called `topics.ListTopics`, I think it might read better as `topics.List`
```suggestion
// List returns a slice of all the Topics
func List(ctx context.Context, client *kafkago.Client) (topics []kafkago.Topic, err error) {
```

Reply:
Done